app/views/tags/index.html.erb

```diff
@@ -19,25 +19,52 @@
   <meta name="twitter:image:src" content="<%= SiteConfig.main_social_image %>">
 <% end %>
 
-<div class="user-profile-header tag-header bg-transparent">
-  <div class="tag-or-query-header-container">
+<style>
```

Comment:
Any reason for the `<style />` tag addition? Is this inline critical CSS?

Reply:
Not critical, but useful :D I've done little "redesign" since yesterday, so we should not need it anymore.